specs/protocol/configurability.md

```diff
@@ -0,0 +1,78 @@
+# OP Stack Configurability
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Consensus Parameters](#consensus-parameters)
+- [Admin Roles](#admin-roles)
+- [Service Roles](#service-roles)
+- [Policy Parameters](#policy-parameters)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+When deploying the OP Stack software to a production environment,
+certain parameters about the protocol can be configured. These
+configurations can include a variety of parameters which affect the
+resulting properties of the blockspace in question.
+
+There are four categories of OP Stack configuration options:
+
+- **Consensus Parameters**: Parameters and properties of the chain that may
+  be set at genesis and fixed for the lifetime of the chain, or may be
+  changeable through a privileged account.
+- **Policy Parameters**: Parameters of the chain that can be changed without breaking consensus. Consensus is enforced while policy includes the set of possible behaviors within consensus. Can be thought of as a superset of consensus.
```

Comment:
Perhaps remove the superset sentence because policy is constrained by consensus, meaning that its a subset. One example of a block builder policy would be "include only 1 tx per sender per block" where consensus allows an arbitrary number of txs per sender per block